backend/config.js

```diff
@@ -9,8 +9,10 @@ const configData = envSchema({
   schema: S.object()
     .prop("AIRTABLE_API_KEY", S.string())
     .prop("AIRTABLE_BASE_ID", S.string())
+    .prop("APP_DOMAIN", S.string().default("localhost").required())
```

Comment:
either add a default value or set it required, but not both, this is usually an antipattern.